readme.md

```diff
@@ -35,11 +35,11 @@ GitHub username.
 
 #### token
 
-Type: `string`  
```

Comment:
As explained earlier, don't change it. The two spaces have semantic value in Markdown. It's a linebreak.